### PR TITLE
[C-1062] Update loading states for favorites and collection page tables

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Quick one to fix some loading states for favorites page and collection page

### Dragons

N/A

### How Has This Been Tested?

Manually tested

### How will this change be monitored?

N/A

### Feature Flags ###

N/A

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/kj-Update-new-table-loading-states

File: packages/common/src/store/pages/saved-page/lineups/tracks/reducer.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+import { Status } from 'models'
 import { RESET_SUCCEEDED, stripPrefix } from 'store/lineup/actions'
 import { initialLineupState } from 'store/lineup/reducer'
 import { PREFIX } from 'store/pages/saved-page/lineups/tracks/actions'
 
 const initialState = {
   ...initialLineupState,
-  prefix: PREFIX
+  prefix: PREFIX,
+  status: Status.LOADING
```
Comment (sliptype) [inline packages/common/src/store/pages/saved-page/lineups/tracks/reducer.ts:11]: cc @dylanjeffers cause you just touched the lineup initial status. Is this the best way to do this?

**Review by sliptype — APPROVED** [review]
lgtm!

File: packages/common/src/store/pages/saved-page/lineups/tracks/reducer.ts (hunk 1 above)
Comment (dylanjeffers) [reply packages/common/src/store/pages/saved-page/lineups/tracks/reducer.ts:11]: yeah this seems like a good idea

Comment (Kyle-Shanks) [reply packages/common/src/store/pages/saved-page/lineups/tracks/reducer.ts]: Good call out. thank you both! 🙏

**Review by dylanjeffers — APPROVED** [review]
perfect

**Pull request merged**